server.py

```diff
@@ -37,22 +41,22 @@ def get_root():
     """Root Endpoint for 3scale."""
     return jsonify(
         status='OK',
-        version=VERSION,
+        version=API_VERSION,
         message='Up and Running'
     )
 
 
-@APP.route(f'{ROUTE_PREFIX}/v0/version', methods=['GET'])
+@APP.route(f'{ROUTE_PREFIX}/v{API_VERSION}/version', methods=['GET'])
```

Comment:
If `PATH_PREFIX` is `api`, then the generated route for volume type validation will be -

`api/aiops-volume-type-validation/v0.1/version`

We are missing a leading slash, so this needs to be -
`@APP.route(f'/{ROUTE_PREFIX}/v{API_VERSION}/version', methods=['GET'])`

Reply:
I've added the leading slash to the `ROUTE_PREFIX` when `PATH_PREFIX` is set + updated the root endpoint. Now we get:

No `PATH_PREFIX`:
```python
>>> APP.url_map
Map([<Rule '/v0.1/version' (HEAD, OPTIONS, GET) -> get_version>,
 <Rule '/v0.1/collect' (POST, OPTIONS) -> post_collect>,
 <Rule '/metrics' (HEAD, OPTIONS, GET) -> get_metrics>,
 <Rule '/' (HEAD, OPTIONS, GET) -> get_root>,
 <Rule '/static/<filename>' (HEAD, OPTIONS, GET) -> static>])
```

`PATH_PREFIX` and `APP_NAME` defined:
```python
APP.url_map
Map([<Rule '/api/aiops-volume-type-validation/v0.1/version' (GET, OPTIONS, HEAD) -> get_version>,
 <Rule '/api/aiops-volume-type-validation/v0.1/collect' (OPTIONS, POST) -> post_collect>,
 <Rule '/api/aiops-volume-type-validation/' (GET, OPTIONS, HEAD) -> get_root>,
 <Rule '/metrics' (GET, OPTIONS, HEAD) -> get_metrics>,
 <Rule '/static/<filename>' (GET, OPTIONS, HEAD) -> static>])
```

Reply:
Thanks, works great now!